app/uk/gov/hmrc/breathingspaceifproxy/connector/service/UpstreamConnector.scala

```diff
@@ -106,13 +106,27 @@ trait DownstreamConnector extends HttpErrorFunctions with Logging with UsingCirc
       else if (message.contains(noPeriodIdFound)) PERIOD_ID_NOT_FOUND
       else RESOURCE_NOT_FOUND
 
-    logErrorAndGenUpstreamResponse(NOT_FOUND, response, baseError)
+    logInfoAndGenDownstreamResponse(NOT_FOUND, response, baseError)
   }
 
-  private def logErrorAndGenUpstreamResponse[T](statusCode: Int, message: String, baseError: BaseError)(
+  private def logErrorAndGenDownstreamResponse[T](statusCode: Int, message: String, baseError: BaseError)(
     implicit requestId: RequestId
   ): ResponseValidation[T] = {
     logger.error(s"Error($statusCode) for $requestId. $message")
     Future.successful(ErrorItem(baseError).invalidNec)
   }
+
+  private def logInfoAndGenDownstreamResponse[T](statusCode: Int, message: String, baseError: BaseError)(
+    implicit requestId: RequestId
+  ): ResponseValidation[T] = {
+    logger.info(s"Error($statusCode) for $requestId. $message")
+    Future.successful(ErrorItem(baseError).invalidNec)
+  }
+
+  private def logWarningAndGenDownstreamResponse[T](statusCode: Int, message: String, baseError: BaseError)(
+    implicit requestId: RequestId
+  ): ResponseValidation[T] = {
+    logger.warn(s"Error($statusCode) for $requestId. $message")
+    Future.successful(ErrorItem(baseError).invalidNec)
+  }
```

Comment:
Again, feels like unnecessary duplication here where all we are changing is the log level used.

Reply:
:+1: